include/alpaka/mem/buf/BufCudaRt.hpp

```diff
@@ -436,8 +435,7 @@ namespace alpaka
                                 &pitchBytes,
                                 widthBytes,
                                 height));
-                        assert(memPtr);
-                        assert(pitchBytes>=widthBytes);
+                        assert(pitchBytes>=widthBytes||(width*height)==0);
```

Comment:
Do you know which pitch cudaMallocPitch returns if the width = 0?
If the pitch is also 0, the original assert would be enough and would not need to be extended.